### PR TITLE
Update swc crates

### PR DESCRIPTION
**Pull request by deckchairlabs**
This PR updates crates to latest versions where applicable, also dogfoods itself by minifying the final wasm.js output.

### CODE REVIEW

File: swc_wasm/build.js [hunk 1]
```diff
@@ -48,57 +47,78 @@ if (!(await Deno.stat("Cargo.toml")).isFile) {
...
+  `\nconst wasmModule = new WebAssembly.Module(source);\nconst wasmInstance = new WebAssembly.Instance(wasmModule, imports);\nconst wasm = wasmInstance.exports;\n`;
+console.log(init);
 
 log("minifying js");
-const output = Terser.minify(`${source}\n${init}`, {
-  mangle: { module: true },
-  output: {
-    preamble: "//deno-fmt-ignore-file",
-  },
+import { minifySync } from "./pkg/deno_swc.js";
```
Comment (littledivy) [inline swc_wasm/build.js:103]: This won't work unless there is already a previous version of deno_swc lying around. See the Build CI failure - https://github.com/nestdotland/deno_swc/runs/6022229717?check_suite_focus=true

**Review by littledivy — COMMENTED** [review]
Thanks for the PR! Looks good overall, just need to get the CI passing.

File: swc_wasm/build.js (hunk 1 above)
Comment (deckchairlabs) [reply swc_wasm/build.js:103]: Ah yes, very silly. Will resolve this

**Comment by littledivy** [comment]
Done in https://github.com/littledivy/deno_swc/pull/43. Thanks for the PR

**Pull request closed without merging**